.github/workflows/automation.yaml

```diff
@@ -0,0 +1,22 @@
+name: Automation
+
+on:
+  pull_request:
+    # https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull-request-event-pull_request
+    types: [opened, synchronize, reopened, ready_for_review]
+
+jobs:
+  required_linear_ticket:
+    name: Required Linear Ticket
+    if: ${{ github.event.pull_request.user.login != 'dependabot[bot]' && github.event.pull_request.user.login != 'github-actions[bot]' }}
+    runs-on: ubuntu-latest
+    steps:
+      - name: Pull Request Linear Ticket Check Action
+        uses: FatbearLabs/ticket-check-action@v2
```

Comment:
The reason the job is failing is because for this repo, there is no `v2` tag. We can create one temporarily and delete it after because once you squash-merge, the tag will not point to anything. Lemme give it a go.